front/src/utils/consts.js

```diff
@@ -94,6 +94,12 @@ export const GetWeatherModes = {
   DailyForecast: 'dailyForecast'
 };
 
+export const NextcloudTalkStatus = {
```

Comment:
Not sure we want `NextcloudTalkStatus` in global `consts.js` file.

Reply:
I am removing `NextcloudTalkStatus` and changing to existing `RequestStatus`